cylc/flow/cfgspec/workflow.py

```diff
@@ -1635,6 +1635,32 @@ def get_script_common_text(this: str, example: Optional[str] = None):
                        MYITEM = %(item)s
                        MYFILE = /path/to/%(i)03d/%(item)s
 
+                    .. note::
+
+                       As with other Cylc configurations, leading or trailing
+                       whitespace will be stripped, so the following two examples
+                       are equivalent:
```

Comment:
This line is too long for flake8
```suggestion
                       whitespace will be stripped, so the following two
                       examples are equivalent:
```